packages/forms-web-app/src/lib/views.js

```diff
@@ -36,7 +36,6 @@ const VIEW = {
 	APPELLANT_SUBMISSION: {
 		TASK_LIST: 'appeal-householder-decision/task-list',
 		APPEAL_STATEMENT: 'appellant-submission/appeal-statement',
-		APPLICATION_NUMBER: 'appellant-submission/application-number',
```

Comment:
Sorry all the route changes are hard to follow - why is this not needed? Just unused?